tests/pytests/test_aggregate.py

```diff
@@ -979,32 +979,40 @@ def testResultCounter(env):
     #env.expect('FT.AGGREGATE', 'idx', '*', 'FILTER', '@t1 == "foo"').equal([0])
 
 def test_aggregate_timeout():
```

Comment:
Since the setup for this function is relatively long (lots of docs to insert) for a single query test, have you considered using `HELLO 3` command to get RESP3 for the second test? I think we should make the effort to shorten the test time as much as we can

Reply:
Absolutely, I indeed tried this, but getting a `Protocol Error` when running `HELLO 3`.